### DISCUSSION TITLE
Do you have any example testing or making spyOn of ky instance?

### DISCUSSION THREAD

**Discussion by andrestejada** [sindresorhus/ky#434] [Q&A] [answered]
do you have any example trying to made a mock of ky or ky instance , especially when you need to mock the response with mock data, Thank you

**Comment by sholladay** [comment, marked as answer]
In my apps, I tend to use [fetch-mock](https://github.com/wheresrhys/fetch-mock). It works with Ky since we use `fetch()` under the hood.

**Comment by andrestejada** [comment]
May be do you have any example , to know how is the correct way to mocking the response?

**Reply by sholladay** [reply]
Please click the link I provided. It has examples. There is no difference between mocking Ky and mocking fetch.

**Comment by VijethKC** [comment]
Answered here with simple example - https://github.com/sindresorhus/ky/discussions/488
